magpie/alembic/env.py

```diff
@@ -1,10 +1,8 @@
 from __future__ import with_statement
-# noinspection PyUnresolvedReferences
-from alembic import context
+from alembic import context  # noqa: F403
 # from logging.config import fileConfig
 from sqlalchemy.schema import MetaData
-# noinspection PyProtectedMember
-from sqlalchemy.engine import create_engine, Connection, Connectable
+from sqlalchemy.engine import create_engine, Connection, Connectable  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'sqlalchemy.engine'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)